### ISSUE TITLE
Support for Overworld Two.

### ISSUE THREAD

**Issue by MagmaSlime123** [KdotJPG/Simply-Improved-Terrain#26]
Title. Untested, but both mods do change terrain generation quite a bit, I'm guessing they'll either crash or have other issues.

https://www.curseforge.com/minecraft/mc-mods/overworld-two-forge

**Comment by KdotJPG** [comment]
Good idea, and it does appear that they have a related issue: https://github.com/jaskarth/overworld-two/issues/26

While it would require a dedicated custom solution to merge the two terrain shape generators, I agree that there should still be compatibility. That way, users of Overworld Two can still get Simply Improved Terrain's terrain shape in other dimensions, and at least have its feature/structure shape changes in the overworld.

I've added this to #27.

**Comment by MagmaSlime123** [comment]
Thanks.
